Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -332,13 +332,20 @@ internal static async Task RunCoreTests(MinioClient minioClient)
             GetBucketPolicy_Test1(minioClient)
         };
 
+#if NET6_0_OR_GREATER
```

Comment:
Please move this to the helper method and do the switch per platform there.

Reply:
Done